mllib/src/test/scala/org/apache/spark/mllib/clustering/StreamingKMeansSuite.scala

```diff
@@ -133,6 +133,19 @@ class StreamingKMeansSuite extends FunSuite with TestSuiteBase {
     assert(math.abs(c1) ~== 0.8 absTol 0.6)
   }
 
+  test("test setDecayFactor") {
+    val initCenters = Array(Vectors.dense(500.0), Vectors.dense(-500.0))
+    val weights = Array(5.0, 5.0)
+
+    // create model with two clusters
+    val kMeans = new StreamingKMeans()
+      .setK(2)
+      .setDecayFactor(1.0)
+      .setInitialCenters(initCenters, weights)
+    val decayFactor = kMeans.decayFactor
+    assert(kMeans.setDecayFactor(0.0).decayFactor != decayFactor)
```

Comment:
Let's simplify the test.

~~~scala
val kMeans = new StrreamingKMeans()
assert(kMeans.decayFactor === 1.0)
kMeans.setDecayFactor(2.0)
assert(kMeans.decayFactor === 2.0)
~~~

It is also useful to include the JIRA number in the test name, in case people wonder why we have this particular test.